### PR TITLE
Cfd dem examples

### PR DESCRIPTION
**Pull request by tonielgeitani**
This PR migrates the packed bed example of the Wiki to the new documentation using sphinx

### CODE REVIEW

**Review by oguevremont — COMMENTED** [review]
Nice work! I corrected a few typos here and there. I wouldn't start talking about a section until its title, but that's personal taste.

File: doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+Description of the case
+-----------------------
+
+This example simulates air flow through a packing of particles. First, we use Lethe-DEM to fill the bed with particles. Then, we use the VANS solver within Lethe to simulate air flow through the packed bed.
+
+
+Parameter file
+--------------
+
+Lethe simulations are controlled by *parameter files* which possess the extension ``.prm``. This is the default text format of the ParameterHandler class of the deal.ii library from which Lethe derives. For more information on this class, we refer to the `deal.II documentation <https://www.dealii.org/current/doxygen/deal.II/classParameterHandler.html>`_.
```
[the remark this answers is not in the document]
Comment (tonielgeitani) [reply doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst]: I deleted this section as it is already mentioned in the lid driven cavity example

File: doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+Parameter files are made of subsections which describe a portion of the simulation (e.g. physical properties, simulation control). In parameter files, lines starting with ``#`` are comments. Parameters are set using the following syntax:
+
+.. code-block:: text
+
+    set parameter name = value
+
+The syntax is flexible. Parameters do not need to be specified in a specific order, but only within the subsection in which they belong. All parameter subsections are described in the `parameter section <../../../parameters.html>`_ of the documentation.
+
+To set-up the cylindrical packed bed case, we first fill the bed with particles. 
+We first introduce the different sections of the parameter file (packing_in_cylinder.prm) needed to run this simulation. In this example, we are simulating a cylindrical packed bed that has a half length of 0.1 m, and a radius of 0.01 m. We use the GridGenerator::subdivided_cylinder function of Deal.II in order to generate the mesh. The cylinder is divided 16 times in the x direction. The following portion of the DEM parameter file shows the function called:
```
[the remark this answers is not in the document]
Comment (tonielgeitani) [reply doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst]: done

File: doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+.. code-block:: text
+
+    subsection mesh
+        set type                 				= dealii
+        set grid type            				= subdivided_cylinder
+        set grid arguments       				= 16:0.01:0.1
+        set initial refinement   				= 1
+    end
+
+A simulation time of 0.6 s was chosen with a time step of 1e-5. It it important to choose a long enough time as to allow all particles to come to rest. We store the output files generated in the folder output_dem:
```
[the remark this answers is not in the document]
Comment (tonielgeitani) [reply doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst]: done

File: doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+
+
+Location of the example
+------------------------
+ ``/examples/cfd-dem/cylindrical_packed_bed``
+
+
+Description of the case
```
Comment (blaisb) [inline doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst:19]: Maybe seperate clearly the DEM initialization and the VANS solver part? Like explain the simulation in two parts.

**Review by blaisb — COMMENTED** [review]
Very minor stuff, soon good to go :)

**Review by oguevremont — APPROVED** [review]

**Review by blaisb — APPROVED** [review]

**Pull request merged**